openshift-oauth-proxy/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceaccount }}
```

Comment:
I think the SA need an annotation: `serviceaccounts.openshift.io/oauth-redirectreference.primary`